core/src/test/java/projectzero/TestMain.java

```diff
@@ -0,0 +1,12 @@
+package projectzero;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
```

Comment:
I'm not to familiar with Jupiter testing. Is this better than JUnit4 library? Just curious.